javascript/js-in-the-real-world/webpack.md

```diff
@@ -4,5 +4,6 @@ In our last lesson we covered the first half of what webpack can do for you - bu
 
 1. Go through the "Core Concepts" lessons at [Webpack Academy](https://webpack.academy/p/the-core-concepts). It starts off with the concepts we've already learned, but do the whole thing. The review is valuable and this presenter explains things very clearly. Make sure you take note of the `--watch` feature!
 2. Go through [this tutorial](https://webpack.js.org/guides/asset-management/) to see examples of using webpack to manage your website's assets.
-3. Finally, do [this tutorial](https://webpack.js.org/guides/output-management/) to learn how to let webpack manually manage your index.html and insert your bundle into the page for you!
-4. You don't need to do the rest of the webpack tutorials at this time, but take a look at what's offered on the sidebar of their [guides page](https://webpack.js.org/guides/). There are several sweet features that you might want to use in future projects such as code-splitting, lazy-loading, and tree-shaking. Now that you have a handle on webpack's configuration system adding these features is as easy as using the right plugins and loaders!.
+3. Read through [this tutorial](https://webpack.js.org/guides/output-management/) to learn how to let webpack manually manage your index.html and insert your bundle into the page for you!
+4. Finally, the first part of [this tutorial](https://webpack.js.org/guides/development/) talks about source maps, a handy way to track down which source file (index.js, a.js, b.js) an error is coming from when you use wepback to bundle them together. It also walks through an example of the `--watch` feature you *definitely* took notes on above.
```

Comment:
I think it would be good to add a something about console/debugging!  Maybe add something like "Enabling source maps is essential to debugging bundled code using your browsers DevTools" after the first sentence.

As for the last sentence, I think better wording would be: "It also walks through an example of the `--watch` feature you *definitely* should have taken note of above."

Reply:
Good thoughts, I'll get those updated!